apps/src/templates/sectionProgressV2/LevelDataCell.jsx

```diff
@@ -138,7 +138,11 @@ function LevelDataCell({
         openInNewTab
         external
         onClick={levelClickedAmplitude(sectionId, level.kind === 'assessment')}
-        className={classNames(styles.expandedLevelLink, linkClassName)}
+        className={classNames(
+          styles.expandedLevelLink,
+          linkClassName,
+          'ui-test-' + level.levelNumber
```

Comment:
I thought a lot about this before this PR - I was nervous to use an id because all the id for levels would be the same (ex. If you opened lesson 2, level 1, the id of that cell would be the same as the cell if you opened lesson 3, level 1).  I didn't want to have duplicate id names.  I went with class names because it would at least be not "guaranteed" to have different names.  Additionally, the lesson number is not passed through already in the data to the component, and the `level.id` is hard for the test to see... THUS I landed on just using the class identifier. 

However, I know that the id is probably makes more sense to a general reader - even if they don't know ALL the reasons why it wasn't given an ID to start.  For that reason, I decided to pass in the `level.path` to the id.  This makes it a lot longer of a name... which folks might question the need of... but it is descriptive, and it is stored in the ID (vs className).